pywinauto/macos/ax_element_info.py

```diff
@@ -84,52 +100,43 @@ def _cf_number_to_py_number(number_value):
         return cf_type_to_py_type[cf_attr_type](attrValue)
     except KeyError:
         # did not get a supported CF type. Move on to AX type
-        ax_attr_type = AXValueGetType(attrValue)
         try:
-            extracted_str = re.search('{.*}', attrValue.description()).group()
-            # obj = ax_type_from_string[ax_attr_type]
-            # t = obj(attrValue)
-            return tuple(ax_type_from_string[ax_attr_type](extracted_str))
-            # return tuple(d[ax_attr_type](attrValue))
-        except KeyError:
-            raise NotImplementedError("Type conversion for {} and {} is not implemented".format(cf_attr_type, ax_attr_type))
-
-
-ax_error_description = {
-    kAXErrorAPIDisabled: 'Assistive applications are not enabled in System Preferences',
-    kAXErrorActionUnsupported: 'The referenced action is not supported',
-    kAXErrorAttributeUnsupported: 'The referenced attribute is not supported',
-    kAXErrorCannotComplete: 'A fundamental error has occurred, such as a failure to allocate memory during processing',
-    kAXErrorFailure: 'A system error occurred, such as the failure to allocate an object',
-    kAXErrorIllegalArgument: 'The value received in this event is an invalid value for this attribute',
-    kAXErrorInvalidUIElement: 'The accessibility object received in this event is invalid',
-    kAXErrorInvalidUIElementObserver: 'The observer for the accessibility object received in this event is invalid',
-    kAXErrorNoValue: 'The requested value or AXUIElementRef does not exist',
-    kAXErrorNotEnoughPrecision: 'Not enough precision',
-    kAXErrorNotImplemented: 'The function or method is not implemented',
-    kAXErrorSuccess: 'No error occurred',
-}
-
-
-class AXError(Exception):
-
-    def __init__(self, err_code):
-        self.err_code = err_code
-        self.message = ax_error_description[err_code]
-
-
-class AxElementInfo(object):
+            # AXValueGetValue is not yet supported, it requires a manual wrapper.
+            # Implement macos structures and update the _cg_val_to_py_obj!!
+            # Think about axstructures
+            return _cg_val_to_py_obj(attrValue)
 
+        except KeyError:
+            raise NotImplementedError("Type conversion for {} and {} is not implemented".format(str(cf_attr_type), str(ax_attr_type)))
+
+class AxElementInfo(ElementInfo):
+
+    re_props = ["class_name", "name", "placeholder", "control_type","subrole"]
+    exact_only_props = ["pid", "visible", "enabled", "rectangle", "description"]
```

Comment:
Check if we can search by `"rectangle"` on Windows. Maybe move `"description"` to `re_props`.